src/controllers/user.controls.js

```diff
@@ -22,22 +22,18 @@ module.exports = {
     createUser: async(req, res, next) => {
         try {
             const user = await uerRepository.createUser(req.body);
-            if(user.username) res.status(201).json(user);
-            else{
-                res.status(409).json({"message":user});
-            }
+            res.status(201).json(user);
         } catch (error) {
-            res.status(409).send(error);
+            res.status(409).send({ error: error });
```

Comment:
why 409? that's oddly specific.
For now, I would just use `400` or `500`.
The convention is that 400 errors mean the client (e.g. your app) sent a bad request. 500 means something went wrong on the server

See more on http status codes here https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses